examples/auto-scaling-groups/destroy.pp

```diff
@@ -0,0 +1,21 @@
+cloudwatch_alarm { ['AddCapacity', 'RemoveCapacity']:
+  ensure => absent,
+  region => 'sa-east-1',
+} ~>
+
+ec2_scalingpolicy { ['scaleout', 'scalein']:
```

Comment:
That rationale here was that Autoscaling is part of the EC2 (customer facing) service, rather than a separate service. For instance the AWS dashboard lists Autoscaling as part of the EC2 options.

My gut feeling here was that going with the user facing name, rather than the API implementation, was the better bet.